tests/qos/files/qos_params.th.yaml

```diff
@@ -508,3 +508,258 @@ qos_params:
                 cell_size: 208
             hdrm_pool_wm_multiplier: 4
             cell_size: 208
+        topo-t0-64:
```

Comment:
@XuChen-MSFT Is this configuration adaptable for all platforms of t0-64?

Reply:
only 6100 T0 use t0-64